tests/probes_test.go

```diff
@@ -47,11 +47,15 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			},
 		}
 		table.DescribeTable("should succeed", func(readinessProbe *v12.Probe, serverStarter func(vmi *v12.VirtualMachineInstance, port int)) {
+			if tests.IsRunningOnKindInfra() && readinessProbe == httpProbe {
+				Skip("Skip http probe test till card https://issues.redhat.com/browse/CNV-4454 is fixed")
```

Comment:
is that public accessable?

Reply:
no, will open a bugzilla, will change it to a PR link. Thanks for noticing it.